test/integration/test_webrtc_api.py

```diff
@@ -0,0 +1,233 @@
+"""
+    Bandwidth
+
+    Bandwidth's Communication APIs  # noqa: E501
+
+    The version of the OpenAPI document: 1.0.0
+    Contact: letstalk@bandwidth.com
+    Generated by: https://openapi-generator.tech
+"""
+
+
+from array import array
+import os
+import unittest
+from urllib import response
+
+import bandwidth
+from hamcrest import *
+from bandwidth.api import participants_api, sessions_api
+from bandwidth.exceptions import NotFoundException, UnauthorizedException
+from bandwidth.model.create_participant_request import CreateParticipantRequest
+from bandwidth.model.create_participant_response import CreateParticipantResponse
+from bandwidth.model.publish_permissions_enum import PublishPermissionsEnum
+from bandwidth.model.device_api_version_enum import DeviceApiVersionEnum
+from bandwidth.model.participant import Participant
+from bandwidth.model.session import Session
+from bandwidth.model.subscriptions import Subscriptions
+from bandwidth.model.participant_subscription import ParticipantSubscription
+
+
+class TestSessionsApi(unittest.TestCase):
```

Comment:
Does this API return proper 400/401/403 errors? In the other tests we created unauthorized and forbidden clients to test the 4xx responses - we should test these 4xx responses too

Reply:
There were already tests in there for a 401 and 404 error, but I couldn't find a way to force a 403 or 400. Most stuff in the api is optional and gets ignored if formatted incorrectly, and if you do something horrendously wrong, you get a 500. If you know how to force either of those lmk and I'll add the test.

Reply:
I dont know how to force those - just wanted to make sure it wasnt overlooked - but sounds like youve covered the bases we have available.